perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -164,21 +205,22 @@ static void SimulateInstruction(const instruction& Instruction, s16* Registers,
                     }
                     else
                     {
+                        LeftOperandValue = Registers[Dest.Register.Index];
                         switch (Instruction.Op)
                         {
                             case Op_add:
-                            {
-                                Registers[Dest.Register.Index] += Value;
-                                Result = Registers[Dest.Register.Index];
+                            {                           
+                                Result = LeftOperandValue + RightOperandValue;
+                                Registers[Dest.Register.Index] = Result;
                             } break;
                             case Op_sub:
                             {
-                                Registers[Dest.Register.Index] -= Value;
-                                Result = Registers[Dest.Register.Index];
+                                Result = LeftOperandValue - RightOperandValue;
+                                Registers[Dest.Register.Index] = Result;
                             } break;
                             case Op_cmp:
                             {
-                                Result = Registers[Dest.Register.Index] - Value;
+                                Result = Registers[Dest.Register.Index] - RightOperandValue;
```

Comment:
You can use `LeftOperandValue` here, it's more readable and it's faster.

Reply:
Missed it while refactoring, thanks
1d77128fb08975415979ce65166529531eb3f676